hello.py

```diff
@@ -1,11 +1,12 @@
-from flask import Flask, Response
+from flask import Flask, render_template
+
 app = Flask(__name__)
-@app.route("/")
+
+@app.route("/", methods=['GET'])
+
 def index():
-    return Response("""
-    <h1 style='color: red;'>HEllO WORLD!</h1>
-    <p>GROOT @inc 2020</p>
-    <code>Flask is <em>awesome</em></code>
-    """), 200
+	return render_template('page.html')
```

Comment:
Where is this file? It seems to be missing in this commit.

Reply:
I separated it to another PR #48  because I thought this is the best practice for PR.

Reply:
A PR should contain an atomic change - meaning, it should contain one change that is complete in itself, changing the repository from a working state to another working state, where there is a single difference between the states. If that means more than one file needs to change, that's fine.

Reply:
we "forced" the separation of the files into different PRs as instructed (even if I agree it could have been a single PR) this was done to minimize the sizes of the PRs as much as possible, the file itself is found on another PR called issue#30 HTML file, can this be merged as is or do we need to commit the HTML file and the templates folder adding them to this PR?